src/cmd/vm.rs

```diff
@@ -379,18 +387,38 @@ fn run_acloudw_cmd(opts: &[&str]) -> Result<()> {
     let acloudw_cmd = opts.join(" ");
     info!("Running `{acloudw_cmd}`...");
     let mut cmd = Command::new("bash")
+        .stdout(Stdio::piped())
         .arg("-c")
         .arg(acloudw_cmd)
         .spawn()
         .context("Failed to execute acloudw")?;
 
+    let buf_reader = BufReader::new(cmd.stdout.take().context("error")?);
+    let port = get_acloudw_port_number(buf_reader)?;
+
     let result = cmd.wait().context("Failed to wait for acloudw")?;
 
     if !result.success() {
         error!("acloudw failed")
     }
 
-    Ok(())
+    Ok(port)
+}
+
+fn get_acloudw_port_number(r: impl BufRead) -> Result<String> {
+    let mut port = String::new();
+    let split_iter = r
+        .split(b'\n')
+        .map(|l| String::from_utf8_lossy(&strip_ansi_escapes::strip(l.unwrap())).to_string());
```

Comment:
Use `strip_ansi_escapes` crate to remove ANSI escape sequences (color code) from output. 

https://docs.rs/strip-ansi-escapes/0.2.0/strip_ansi_escapes/

Reply:
I think it is a good idea to add a comment in the code to explain why removing the ANSI escape sequence here is needed (like `Remove the escape sequences here to make the regexp search work` or something)

Reply:
Thank you for the advice! 
I added a comment.